app/code/Magento/Paypal/Block/Iframe.php

```diff
@@ -116,7 +116,7 @@ protected function _construct()
             if ($file && $directory->isExist($directory->getRelativePath($file))) {
                 $this->setTemplate($templateFile);
             } else {
-                $this->setTemplate('hss/iframe.phtml');
+                $this->setTemplate('Magento_Paypal::hss/iframe.phtml');
```

Comment:
Please, revert fix in places when the template specified dynamically

Reply:
Done.